rebalance/rebalance.py

```diff
@@ -625,14 +664,27 @@ def init(options, configuration, plugin):
     plugin.maxhops = int(options.get("rebalance-maxhops"))
     plugin.msatfactor = float(options.get("rebalance-msatfactor"))
     plugin.erringnodes = int(options.get("rebalance-erringnodes"))
+    getroute_switch = {
+        "basic": getroute_basic,
+        "default": getroute_default
+    }
+    plugin.getroute = getroute_switch.get(options.get("rebalance-getroute"), getroute_default)
 
     plugin.log(f"Plugin rebalance initialized with {plugin.fee_base} base / {plugin.fee_ppm} ppm fee  "
                f"cltv_final:{plugin.cltv_final}  "
                f"maxhops:{plugin.maxhops}  "
                f"msatfactor:{plugin.msatfactor} "
-               f"erringnodes:{plugin.erringnodes}")
+               f"erringnodes:{plugin.erringnodes} "
+               f"getroute: {plugin.getroute.__name__}")
 
 
+plugin.add_option(
+    "rebalance-getroute",
+    "default",
+    "Getroute method for route search"
+    "Can be 'default' or 'basic' for simple search"
```

Comment:
We should point out the differences here a bit and maybe rename default to iterative. Like:
```
"Can be 'basic' or 'iterative'. "
"'basic': Tries all routes sequentially." 
"'iterative': Tries shorter and bigger routes first."
```